contracts/core/VolOracle.sol

```diff
@@ -19,22 +19,24 @@ contract VolOracle is DSMath {
      * Immutables
      */
     uint32 public immutable period;
+    uint256 public immutable windowSize;
     uint256 public immutable annualizationConstant;
     uint256 internal constant commitPhaseDuration = 1800; // 30 minutes from every period
 
     /**
      * Storage
      */
     struct Accumulator {
-        // Max number of records: 2^16-1 = 65535.
-        // If we commit twice a day, we get to have a max of ~89 years.
-        uint16 count;
+        // Stores log-return observations over window
+        int256[] observations;
+        // Stores the index of next observation
+        uint8 currObv;
         // Timestamp of the last record
         uint32 lastTimestamp;
         // Smaller size because prices denominated in USDC, max 7.9e27
         int96 mean;
-        // Stores the sum of squared errors
-        uint112 m2;
+        // Stores the dsquared (variance * count)
+        uint112 dsq;
```

Comment:
We can increase the size from uint112 to uint120 for `dsq` since we removed `count`

Reply:
I don't see this updated to uint120 yet